web/styles/dark_theme.css

```diff
@@ -319,6 +319,25 @@
     .help_link_widget:hover {
         color: inherit;
     }
+
+    #uppy-editor .uppy-ImageCropper {
+        /* This CSS is to handle cropper canvas in dark theme as we are not using uppy dashboard.
+
+        Link for source - https://github.com/transloadit/uppy/blob/b7605df9402e4db9b02828bf24558e21eae76637/packages/%40uppy/image-editor/src/style.scss#L123
+        and https://github.com/transloadit/uppy/blob/b7605df9402e4db9b02828bf24558e21eae76637/packages/%40uppy/image-editor/src/style.scss#L133 */
+        .cropper-modal {
+            opacity: 0.7;
+            background-color: hsl(0deg 0% 0%);
+        }
+
+        .cropper-view-box {
+            background: repeating-conic-gradient(
+                    hsl(0deg 1.18% 16.67%) 0% 25%,
+                    hsl(0deg 0% 0%) 0% 50%
+                )
+                50%/16px 16px;
+        }
+    }
```

Comment:
This seems substantially less messy than using Dashboard and hiding it. But I don't understand the comment -- I think more explanation would be helpful. Like, it's not at all obvious without clicking those code links why using Uppy Dashboard might be relevant at all. An explanation of the form "Uppy's cropper canvas has ... which doesn't work with the dark theme because ... When using the Uppy Dashboard plugin, this is solved by ... . However, we're not using that plugin, so we need to provide our own dark theme CSS for this detail."

I also wonder if there's an issue we should be filing upstream for this; is it a bug we're working around?

Reply:
OK. I can live with this.